iam-policy-secretsmanager.tf

```diff
@@ -30,7 +30,24 @@ resource "aws_iam_role_policy" "secretsmanager_for_ecs_task_readwrite" {
     "Statement": {
         "Effect": "Allow",
         "Action": "secretsmanager:GetSecretValue",
-        "Resource": [ "${join("\",\"",concat(data.aws_secretsmanager_secret.db_rw_user.*.arn,data.aws_secretsmanager_secret.ldap_user.*.arn,data.aws_secretsmanager_secret.ranger_audit.*.arn))}" ]
+        "Resource": [ "${join("\",\"",concat(data.aws_secretsmanager_secret.db_rw_user.*.arn,data.aws_secretsmanager_secret.ldap_user.*.arn,data.aws_secretsmanager_secret.ranger_audit.*.arn,data.aws_secretsmanager_secret.docker_registry.*.arn))}" ]
```

Comment:
same here, container wont need to read docker credentials?

Reply:
removed this. updated PR